CHANGELOG.md

```diff
@@ -378,3 +378,5 @@ Ensure output dir exists in `#artifact_path` method.
 [@Vasfed]: https://github.com/Vasfed
 [@szemek]: https://github.com/szemek
 [@mkldon]: https://github.com/mkldon
+[@dmagro]: https://github.com/dmagro
```

Comment:
Let's add also a change log entry under `master` section in this document

Reply:
@palkan can you check it?

Reply:
👍